src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -68,7 +68,7 @@ public static void inferEvents(Properties props, String species) throws Exceptio
 		dbAdaptorPrev = new MySQLAdaptor(host, prevDatabase, username, password, port);
 		if (dbAdaptor == null || dbAdaptorPrev == null) {
 			logger.fatal("Null MySQLAdaptor, terminating orthoinference");
-			return;
+			System.exit(-1);
```

Comment:
I think exit codes are typically only supposed to be 0-255 (https://www.shellscript.sh/exitcodes.html).  Maybe 1 instead of -1 to indicate an error?

Reply:
Changed the -1 to 1